tests/orchestrator/test_retries.py

```diff
@@ -123,12 +123,9 @@ def _schedule_events(context: ContextBuilder, id_counter: int) -> Tuple[ContextB
         Tuple[ContextBuilder, int, List[int]]:
             The updated context, the updated counter, a list of event IDs for each scheduled event
         """
-        scheduled_ids: List[int] = []
-        for id_ in range(num_activities):
-            scheduled_ids.append(id_)
-            context.add_task_scheduled_event(name='Hello', id_=id_)
-            id_counter += 1
-        return context, id_counter, scheduled_ids
+        id_counter = id_counter + 1
```

Comment:
Is there a reason we moved these tests to longer use a for-loop worth of activities?

Reply:
We now do this later in the code, around line `216`, as follows:

```Python
    for city in CITIES:
        # Schedule the events
        context, id_counter = _schedule_events(context, id_counter)
        context.add_orchestrator_completed_event()
```

I think I changed this for readability :)